### PR TITLE
Re-write the use section to remove unattributed examples

### PR DESCRIPTION
**Pull request by doomspork**
It was brought to my attention by @p-buse that we're including a portion of someone else's blog post and that it was unattributed.  Since we avoid referencing and linking to third-party sites, we need to remove or update this section which this PR seeks to do.

I'd love some thoughts.

### CODE REVIEW

File: en/lessons/basics/modules.md [hunk 1]
```diff
@@ -209,47 +209,70 @@ If we attempt to call a macro that is not yet loaded Elixir will raise an error.
...
 
-The Phoenix framework makes use of use and `__using__/1` to cut down on the need for repetitive alias and import calls in user defined modules.
+```elixir
+iex> Example.hello("Sean")
+"Hi, Sean"
+```
 
-Here’s an nice and short example from the Ecto.Migration module:
+Here we can see that `use` invoked the `__using__/1` callback on `Hello` which in turn added the resulting code to our module.
+Now that we've demostrated a basic example let's update our code to look at how `__using__/1` supports options.
```
Comment (nscyclone) [inline en/lessons/basics/modules.md]: `demostrated` → `demonstrated`

File: en/lessons/basics/modules.md [hunk 2]
```diff
@@ -209,47 +209,70 @@ If we attempt to call a macro that is not yet loaded Elixir will raise an error.
...
+Let's update our `Example` module to include the newly created `greeting` option:
+
+
+```elixir
+defmodule Example do
+  use Hello, greeting: "Hola"
+end
+``
+
+If we give it a try in IEx we should see that the greeting is has changed:
```
Comment (nscyclone) [inline en/lessons/basics/modules.md]: `is has` → `has been`

File: en/lessons/basics/modules.md [hunk 3]
```diff
@@ -209,47 +209,70 @@ If we attempt to call a macro that is not yet loaded Elixir will raise an error.
...
 
-The `Ecto.Migration.__using__/1` macro includes an import call so that when you `use Ecto.Migration` you also `import Ecto.Migration`. It also sets up a module property which we will assume controls Ecto’s behavior.
+Let's update our `Example` module to include the newly created `greeting` option:
+
+
+```elixir
+defmodule Example do
+  use Hello, greeting: "Hola"
+end
+``
```
Comment (nscyclone) [inline en/lessons/basics/modules.md]: It looks like a third backtick is needed here.

**Comment by doomspork** [comment]
Thanks @nscyclone!  I've incorporated you feedback.  Any other thoughts?

**Pull request merged**